dom/src/DOMSource.ts

```diff
@@ -7,7 +7,10 @@ export interface EventsFnOptions {
 
 export interface DOMSource {
   select(selector: string): DOMSource;
-  elements(): MemoryStream<Document | Element | Array<Element> | string>;
+  elements(): MemoryStream<
+    Document[] | HTMLBodyElement[] | Array<Element> | string
```

Comment:
I would change `Array<Element>` to `Element[]`. Reads a lot better

Reply:
We use `Array<T>` everywhere else in the codebase, we should stay consistent in that

Reply:
Rhen we should adjust the other ones too

Reply:
like `Document[]`

Reply:
It's in a lot of places throughout the whole monorepo and I'm not convinced `T[]` is just "better"

Reply:
Or do you mean in that line of code there? :D
I think I missed that